src/stud.cpp

```diff
@@ -0,0 +1,25 @@
+ #include "stud.hpp"
+ #include <iostream>
+ 
+ Student::Student()
+         :Student {"name","",""}{
+         }
+ Student::Student(std::string nameVal, std::string surnameVal, std::string       discordNameVal)
```

Comment:
Students data should be different. There is no link to GitHub or Firecode or group.